src/main/java/org/opentripplanner/middleware/utils/HttpUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.opentripplanner.middleware.utils;
+
+import org.eclipse.jetty.http.HttpStatus;
+import spark.Request;
+
+import static org.opentripplanner.middleware.utils.JsonUtils.logMessageAndHalt;
+
+public class HttpUtils {
+
+    /**
+     * Get entity attribute value from request.
+     */
+    public static String getParamFromRequest(Request req, String paramName, boolean allowNull) {
```

Comment:
Can we rename this to `getRequiredParamFromRequest` and add a note that this method will halt the request if the param is not provided?

Reply:
Updated.